src/cljs/zetawar/events/ui.cljs

```diff
@@ -210,6 +213,21 @@
                   :action/r r}]
                 [::clear-selection]]}))
 
+(defmethod router/handle-event ::repair-targeted
+  [{:as handler-ctx :keys [db]} _]
+  (let [game (app/current-game db)
+        cur-faction-color (game/current-faction-color game)
+        [repairer-q repairer-r] (app/selected-hex db)
+        [wounded-q  wounded-r]  (app/targeted-hex db)]
```

Comment:
Just a style comment, but I'd remove the extra whitespace here. I *mostly* don't try to align things in let statements. I like the way it looks, but it's just too hard to be consistent about it imo.